migrations/20211201111510_add-context-column-to-group-widgets.js

```diff
@@ -0,0 +1,32 @@
+
+exports.up = async function (knex) {
+  await knex.schema.table('group_widgets', t => {
+    t.string('context')
+  })
+
+  const now = new Date().toISOString()
+
+  await knex.raw(`
+  INSERT INTO "public"."widgets"("id","name","created_at") VALUES
+    (15,E'opportunities_to_collaborate','${now}'),
+    (16,E'farm_map','${now}'),
```

Comment:
same here, hold off on new widgets until we are ready for them? but if these is easier for testing then its fine

Reply:
I think these are all fine.